internal/services/zero_trust_access_identity_provider/resource_test.go

```diff
@@ -461,7 +461,7 @@ func TestAccCloudflareAccessIdentityProvider_SCIM_Secret_Enabled_After_Resource_
 				ImportState:             true,
 				ImportStateVerify:       true,
 				ImportStateIdPrefix:     fmt.Sprintf("accounts/%s/", accountID),
-				ImportStateVerifyIgnore: []string{"config.client_secret"},
+				ImportStateVerifyIgnore: []string{"config.client_secret", "config.conditional_access_enabled"},
```

Comment:
I don't think this should be necessary now that we're setting the value to false in the config?